model/README.md

```diff
@@ -196,6 +199,76 @@ model in augmented reality to allow the user to visualize it at real scale in th
 surroundings. To support this, new DOM APIs may be added or the existing HTML Fullscreen API extended
 via more [FullscreenOptions](https://fullscreen.spec.whatwg.org/#dictdef-fullscreenoptions) properties.
 
+### Controlling the camera
+
+While by setting the `interactive` property the author will allow a built-in behavior such that dragging
+over a `<model>` element would result in modifying the camera, we propose to allow authors direct control
+of the camera via DOM APIs. An initial proposition would be to add an `HTMLModelElementCamera`:
+
+```idl
+dictionary HTMLModelElementCamera {
+    double pitch;
+    double yaw;
+    double scale;
+};
+```
+
+Then the camera can be set and read back:
+
+```idl
+interface HTMLModelElement : HTMLElement {
+    Promise<HTMLModelElementCamera> getCamera();
+    Promise<undefined> setCamera(HTMLModelElementCamera camera);
```

Comment:
Really setting a viewpoint.

Reply:
3d formats tend to call these cameras even if it doesn't make complete sense here.  (they might also have different properties, like fov/lens)